RHEL6_7/storage/multipath/check

```diff
@@ -23,11 +23,11 @@ if [ -f "$MULTIPATH_CONF" ]; then
 		ERR=1
 	fi
 	if grep -q "devices[[:space:]]*{" $MULTIPATH_CONF; then
-		log_high_risk "Multipath matches user device configs with built-in device configs differently in Red Hat Enterprise Linux 7."
+		log_warning "Multipath matches user device configs with built-in device configs differently in Red Hat Enterprise Linux 7. This will be taken care of by the postupgrade script."
```

Comment:
You see here used the `log_high_risk` originaly with `$RESULT_FIXED` later.  It smells like the unreliable solution and in that case the result fixed should not be used, whether the solution is not reliable. Same for `log_warning` - doesn't make sense to have log_warning combined with `$RESULT_FIXED`

Reply:
Same as the above